src/types.ts

```diff
@@ -67,3 +76,13 @@ export interface NewBidParameters {
   startBlock?: string;
   expireBlock?: string;
 }
+
+export interface SetBuyNowParams {
+  amount: string;
+  tokenId: string;
+}
+export interface BuyNowParams extends SetBuyNowParams {
```

Comment:
None of these additional parameters are needed; they aren't used anywhere.

Reply:
Consider removing this interface, and then renaming `SetBuyNowParams` to just `BuyNowParams` since it's the parameters for both BuyNow/SetBuyNow